lib/cocoapods-packager/builder.rb

```diff
@@ -174,7 +174,15 @@ def static_libs_in_sandbox(build_dir = 'build')
     end
 
     def xcodebuild(defines = '', args = '', build_dir = 'build')
-      `xcodebuild #{defines} CONFIGURATION_BUILD_DIR=#{build_dir} clean build #{args} -configuration Release -target Pods -project #{@sandbox_root}/Pods.xcodeproj 2>&1`
+      command = "xcodebuild #{defines} CONFIGURATION_BUILD_DIR=#{build_dir} clean build #{args} -configuration Release -target Pods -project #{@sandbox_root}/Pods.xcodeproj 2>&1"
+      output = `#{command}`.lines.to_a
+
+      if $?.exitstatus != 0
+        puts "Build command failed: #{command}"
+        puts "Output:"
```

Comment:
Let's wrap these two into something like this <https://github.com/CocoaPods/CocoaPods/blob/546b1252802d04b3a4b111aae90661f2bb9c4c82/lib/cocoapods/command.rb#L59> and this should be good to merge!